### PR TITLE
Implement `dropAllUsersFromDatabase` command

### LINKED ISSUE #1492: Implement `dropAllUsersFromDatabase` command
### What should be done?

With `usersInfo` implemented, we should implement the basic [`dropAllUsersFromDatabase ` command](https://www.mongodb.com/docs/manual/reference/command/dropAllUsersFromDatabase/).

Tests should use `createUser` and `usersInfo` commands, not `insert` and `delete`.

### Where?

https://github.com/FerretDB/FerretDB/tree/main/internal/handler

### Definition of Done

- handler updated;
- integration/compatibility tests added/updated;
- spot refactorings done.

### PR DESCRIPTION
**Pull request by henvic**
# Description

Implement dropAllUsersFromDatabase command.

Closes #1492.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/users/drop_all_users_from_database_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+	db := collection.Database()
+	client := collection.Database().Client()
+	users := client.Database("admin").Collection("system.users")
+
+	if setup.IsMongoDB(t) {
+		assert.NoError(t, collection.Database().RunCommand(ctx, bson.D{
+			{"dropAllUsersFromDatabase", 1},
```
Comment (AlekSi) [inline integration/users/drop_all_users_from_database_test.go:43]: Well, we now implemented that command, so I guess we could do that for both FerretDB and MongoDB. Also, we can do that in other tests too.

File: internal/handler/msg_dropallusersfromdatabase.go [hunk 2]
```diff
@@ -29,6 +34,64 @@ func (h *Handler) MsgDropAllUsersFromDatabase(ctx context.Context, msg *wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/1492
-	return nil, common.Unimplemented(document, document.Command())
+	// NOTE: In MongoDB, the comment field isn't saved in the database, but used for log and profiling.
```
Comment (AlekSi) [inline internal/handler/msg_dropallusersfromdatabase.go:37]: Please port comments from previous PRs

File: integration/users/drop_all_users_from_database_test.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	assert.NoError(t, err)
+
+	actual := integration.ConvertDocument(t, res)
+	actual.Remove("$clusterTime")
+	actual.Remove("operationTime")
+
+	expected := must.NotFail(types.NewDocument("n", int32(quantity), "ok", float64(1)))
+	testutil.AssertEqual(t, expected, actual)
+
+	assert.Equal(t, mongo.ErrNoDocuments, users.FindOne(ctx, bson.D{{"db", db.Name()}}).Err())
```
Comment (rumyantseva) [inline integration/users/drop_all_users_from_database_test.go:77]: The collection to store users is global for all the databases, right? 
Isn't this test flaky having in mind that there is a separate test to create users?
E.g. if a test to create users is run in parallel with a test to drop users, there is a probability that this one will return some data (if a user was just created in the test to create users).

Comment (rumyantseva) [reply integration/users/drop_all_users_from_database_test.go:77]: ah, got it! We always filter by the database - bo th here and when we drop users. Thanks for the explanation!

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by noisersup — APPROVED** [review]

**Comment by AlekSi** [comment]
@henvic https://github.com/FerretDB/FerretDB/blob/fc1163b1f2d4480cbf48df37db683c9740fbc4c4/.github/PROCESS.md?plain=1#L45-L46

**Comment by henvic** [comment]
@AlekSi, thanks!

**Pull request merged**